tests/tools/jira/test_jira_dryrun_client.py

```diff
@@ -0,0 +1,51 @@
+import os
+import pytest
+from unittest.mock import Mock
+from tools.jira_client.jira_api import CLOSED_STATUS
+from tools.jira_client.jira_dryrun_client import JiraDryRunClient
+
+
+def get_devnull_stream():
+    return open(os.devnull, "w")
```

Comment:
```suggestion
@pytest.fixture()
def devnull_stream():
    with open(os.devnull, "w") as devnull:
        yield devnull
```

then tests can just have it as an argument. e.g.:
```python
def test_something(devnull):
...
```